packages/fiori/test/pages/NotificationListItem.html

```diff
@@ -17,26 +17,72 @@
 	<ui5-shellbar id="shellbar" primary-title="Product Title" show-notifications>
 	</ui5-shellbar>
 
-	<ui5-list id="notificationList" header-text="Notifications titleText and content 'truncates'">
+	<ui5-list id="notificationListWithMenu" header-text="Notifications that use Menu for actions as per the new design (2024)">
```

Comment:
I suggest to skip the additional information (soon it wouldn't be new):
Notifications that use Menu for actions